main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -162,7 +162,7 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
   object Declarations {
 
     def Namespace: Rule1[ParsedAst.Declaration.Namespace] = rule {
-      optWS ~ SP ~ (keyword("namespace") | keyword("mod")) ~ WS ~ Names.Namespace ~ optWS ~ '{' ~ UsesOrImports ~ Decls ~ optWS ~ '}' ~ SP ~> ParsedAst.Declaration.Namespace
```

Comment:
Could we restore the Parser; just to give people a bit more time to upgrade themselves. Thanks!